charts/fluent-bit/templates/_helpers.tpl

```diff
@@ -62,6 +62,15 @@ Create the name of the service account to use
 {{- end -}}
 {{- end -}}
 
+{{/*
+Fluent-bit image with tag/digest
+*/}}
+{{- define "fluent-bit.image" -}}
+{{- $tag := ternary "" (printf ":%s" (toString .tag)) (or (empty .tag) (eq "-" (toString .tag))) -}}
+{{- $digest := ternary "" (printf "@%s" .digest) (empty .digest) -}}
```

Comment:
This doesn't look right, as the format for a digest and no tag should be `<repo>:<digest>` but this would return `<repo>@<digest>`.

Reply:
I checked and you're right, my bad.